app/js/home/list-service/tests/list.service.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('ListService (Service test)', function () {
```

Comment:
Do not include "(Service test)", "(Unit test)", etc. in describe. The file name says it already.